pkg/zvol/v1alpha1/zvol.go

```diff
@@ -0,0 +1,140 @@
+package v1alpha1
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"time"
+
+	"github.com/openebs/maya/pkg/util"
+)
+
+// ZVolStatus is zvol's status
+type ZVolStatus string
+
+// ZVolRebuildingStatus is zvol's rebuilding status
+type ZVolRebuildStatus string
```

Comment:
This seems ideal to be put in `pkg/apis/openebs.io/zvol/v1alpha1/`

Reply:
sounds good